### ISSUE TITLE
Is it possible to set seed for tf.train.sgd() ?

### ISSUE THREAD

**Issue by woraperth** [tensorflow/tfjs#316]
To get help from the community, check out our [Google group](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs).

#### TensorFlow.js version
0.11.1

#### Browser version
Google Chrome version Version 66.0.3359.181

#### Describe the problem or feature request
Cannot find a way to set seed in tf.train.sgd. This means I cannot have the same result every time I refresh the page.

#### Code to reproduce the bug / link to feature request

Thank you very much.

Cheers,
Perth
